goonstation.dme

```diff
@@ -1871,6 +1871,7 @@ var/datum/preMapLoad/preMapLoad = new
 #include "code\z_dmm_suite\map_placeholders.dm"
 #include "code\z_dmm_suite\reader.dm"
 #include "code\z_dmm_suite\writer.dm"
+#include "icons\obj\station_map.dm"
```

Comment:
no code inside the icons folder. please mode this to a folder in `modules/`

Reply:
ha